src/broadcast.rs

```diff
@@ -339,6 +339,8 @@ where
     }
 }
 
+impl<'a, T> Unpin for Send<'a, T> {}
```

Comment:
Heh, these kinds of unconditional `Unpin` implementations make my spider senses tingle. It should be correct in how it's used right now. It implies that it would be *incorrect* to perform a pin projection through to the `value` field (which is `T` here). Just something one needs to be mindful of. But ultimately this is all effectively harmless since these types are all private.

I'll consider this closer tomorrow. Thanks for the PR!

Reply:
Right - I only added this with the knowledge that it wouldn't ever be necessary to pin project through the `value` field.